table/format.cc

```diff
@@ -370,8 +370,8 @@ Status UncompressBlockContentsForCompressionType(
     MeasureTime(ioptions.statistics, DECOMPRESSION_TIMES_NANOS,
       timer.ElapsedNanos());
     MeasureTime(ioptions.statistics, BYTES_DECOMPRESSED, contents->data.size());
```

Comment:
MeasureTime is not actually measures time here. Should it also be put outside the if-then block?

Reply:
Good point.

Reply:
It's a histogram though, so expect regression:)